x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/utils.ts

```diff
@@ -44,8 +46,32 @@ export const COMPATIBLE_SPLIT_FIELD_TYPES: DataType[] = [
   KBN_FIELD_TYPES.IP,
 ];
 
+export async function redirectToADJobWizards(
+  embeddable: Embeddable,
+  layerIndex: number,
+  share: SharePluginStart
+) {
+  const { query, filters, to, from, vis } = getJobsItemsFromEmbeddable(embeddable);
+  const locator = share.url.locators.get(ML_APP_LOCATOR);
+
+  const url = await locator?.getUrl({
+    page: ML_PAGES.ANOMALY_DETECTION_CREATE_JOB_FROM_LENS,
+    pageState: {
+      vis: vis as any,
```

Comment:
can you get rid of `any` somehow?

Reply:
When I originally wrote this code, I spent a lot of time attempting to convert the vis type `Readonly<Document>` to a `SerializableRecord` or something similar that will allow it to be used in `pageState`
I gave up.

I'll change this to `vis: vis as unknown as SerializableRecord` which is only slightly better than `any`

Reply:
Changed in https://github.com/elastic/kibana/pull/136421/commits/336db0cd948bdeddc99d02c901e0eb76454e0fc5